### ISSUE TITLE
Document that log() in streams requires a DEBUG logging level

### ISSUE THREAD

**Issue by rucek** [akka/akka#23464]
After just following the [docs for using `log()`](http://doc.akka.io/docs/akka/current/scala/stream/stream-cookbook.html#logging-elements-of-a-stream) the logging "doesn't work", i.e. the logged elements are not visible in the logs. Thisi is because the logging level nedds to be set to `DEBUG` with `akka.loglevel = "DEBUG"`, which the docs fail to mention.

**Comment by ktoso** [comment]
I disagree - the docs show how to change the log level of "per element:

`.withAttributes(Attributes.logLevels(onElement = Logging.WarningLevel))`

if that's unclear that should be clarified, perhaps along with the fact that by default the elements are logged on debug?

**Comment by rucek** [comment]
You're right, I missed it somehow. Then I'd just add a note to clarify that the default level is debug. If you're OK with it, a PR will be there shortly.
